features/cli/cancel_unpaid_orders.feature

```diff
@@ -0,0 +1,26 @@
+@canceling_unpaid_orders
+Feature: Canceling unpaid orders
+    In order to have my orders list free from completed but unpaid orders
+    As a Developer
+    I want to have unpaid orders cancelled
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "PHP T-Shirt" priced at "$10.00" in "United States" channel
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And there is a customer "john.doe@gmail.com" that placed an order "#00000025"
+        And the customer bought a single "PHP T-Shirt"
+        And the customer chose "Free" shipping method to "United States" with "Offline" payment
+
+    @cli
+    Scenario: Having order cancelled after 6 days of being unpaid
+        Given there is a customer "john.doe@gmail.com" that placed an order "#00000026"
+        And the customer bought a single "PHP T-Shirt"
+        And the customer chose "Free" shipping method to "United States" with "Offline" payment
+        And the order "#00000026" is already paid
+        But the customer has refunded the order with number "#00000026"
```

Comment:
What about other states? According to [this configuration](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order_payment.yml#L8-L16) we have 9 possible payment states of the order. We can skip `cancelled` (obvious reasons) and `cart` (it's not even the actual order yet). There are 7 more from which we check only `paid` and `refunded` (or even not so well, see below). I believe. we should cover some more changes regarding [this change](https://github.com/Sylius/Sylius/pull/13660/files#diff-80a4e30faa7db60fa554141928d10a4ff25d1e286a01d0b5ba30facb8ed339a9R318)